s3prl/downstream/asr/hf_hub_templates/README.md

```diff
@@ -0,0 +1,13 @@
+---
+library_name: superb
+benchmark: superb
+task: asr
+tags:
+- superb
+- automatic-speech-recognition
+widget:
+- label: Librispeech sample 1
+  src: https://cdn-media.huggingface.co/speech_samples/sample1.flac
+---
+
+# Test for s3prl push to hub after fine-tuning
```

Comment:
Maybe let's change this to something a bit more "correct":

```suggestion
# Fine-tuned s3prl model for ASR
```

(bonus point if we have the upstream model name in the model card as well, but it could be a follow up PR)